src/test/handlers/_support.js

```diff
@@ -30,8 +30,8 @@ class Database {
 
 const createDb = () => (new Database());
 
-const createServer = (db, logger = () => {}, database) => http.createServer(
-    appCreateServer(config, db, logger, database).callback(),
+const createServer = (db, logger = () => {}, database, configOverride) => http.createServer(
```

Comment:
Probably better to replace this with something like
```suggestion
const createServer = ({ db, logger = () => {}, database, optConfig = config } = {}) => http.createServer(
```

Reply:
(Yeah, you'll have to change the call sites, but it's definitely a nicer pattern)

Reply:
im going for minimum delta every time, unless its a planned refactoring.

Reply:
Piling bad on top of bad is a good way to end up needing a refactor

https://github.com/mojaloop/finance-portal-backend-service/pull/30